jettons/AkitaOnTon.yaml

```diff
@@ -0,0 +1,11 @@
+name: AkitaOnTon
+description: "The cutest dog on TON"
+image: "https://github.com/AkitaOnTon/AkitaOnTon.yml/blob/main/Akita.png"
```

Comment:
could you please update the link so it points to raw.githubusercontent.com?

Reply:
Ready to go